### PR TITLE
fix(cosmetic): cannot find m-r-10 class in superset.less

### LINKED ISSUE #20275: Cannot find .m-r-10 class
## Description

e.g.  Cannot find '.m-r-10' css class in CollectionTable component, seems missing in src/assets/stylesheets/superset.less.

## Design input
[describe any input/collaboration you'd like from designers, and
tag accordingly. For design review, add the
label `design:review`. If this includes a design proposal,
include the label `design:suggest`]

### PR DESCRIPTION
**Pull request by Renderz**
### SUMMARY

Cannot find '.m-r-10' css class in CollectionTable component, seems missing in src/assets/stylesheets/superset.less

BTW, maybe we should check any other missing css class.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

Before:

After:

### ADDITIONAL INFORMATION

- [x] Has associated issue: Fixes #20275
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by rusackas** [comment]
While this fixes the missing class, and I appreciate the contribution to resolve that, there are a couple things that are holding me up from approving/merging.

1) The `.m-r-10` definition exists in SQL Lab's `main.less` file, so if we were to go global with it, we should remove that in favor of the one you're adding.

2) It looks like this class is only actually used in a couple places. It'd probably be better to style those two components with Emotion rather than adding more LESS (which we're trying to whittle away at)

3) To confuse things further... I kind of LIKE the layout of the Before picture better... everything is nicely right aligned. Maybe we should just get rid of this margin once and for all? It sounds like you were solving a code problem more than a cosmetic problem, but let me know if I'm misinterpreting.

**Comment by Renderz** [comment]
> While this fixes the missing class, and I appreciate the contribution to resolve that, there are a couple things that are holding me up from approving/merging.
> 
> 1. The `.m-r-10` definition exists in SQL Lab's `main.less` file, so if we were to go global with it, we should remove that in favor of the one you're adding.
> 2. It looks like this class is only actually used in a couple places. It'd probably be better to style those two components with Emotion rather than adding more LESS (which we're trying to whittle away at)
> 3. To confuse things further... I kind of LIKE the layout of the Before picture better... everything is nicely right aligned. Maybe we should just get rid of this margin once and for all? It sounds like you were solving a code problem more than a cosmetic problem, but let me know if I'm misinterpreting.

I found this problem when I was trying to add another Button on the right side after the one showing in the picture, and there was no margin between these two Button which should be there because both Button had '.m-r-10' class. So I raised up this issue.

But just as you mentioned, it looks nicely right aligned so maybe all those buttons should be removed from '.m-r-10' class? or use '.m-l-10' instead. 

And maybe it is another good way to globalize SQL Labs' less file.

**Comment by michael-s-molina** [comment]
> It looks like this class is only actually used in a couple places. It'd probably be better to style those two components with Emotion rather than adding more LESS (which we're trying to whittle away at)

I think this is the best solution.

> I found this problem when I was trying to add another Button on the right side after the one showing in the picture, and there was no margin between these two Button which should be there because both Button had '.m-r-10' class.

That's a valid use case. We just need to fix it using Emotion instead of LESS.

File: superset-frontend/src/components/Datasource/DatasourceEditor.jsx [hunk 1]
```diff
@@ -123,6 +123,11 @@ const StyledColumnsTabWrapper = styled.div`
   }
 `;
 
+const StyledButtonWrapper = styled.span`
+  margin-top: 10px;
+  margin-left: 10px;
+`;
```
Comment (michael-s-molina) [inline superset-frontend/src/components/Datasource/DatasourceEditor.jsx:129]:
```suggestion
const StyledButtonWrapper = styled.span`
  ${({ theme }) => `
    margin-top: ${theme.gridUnit * 3}px;
    margin-left: ${theme.gridUnit * 3}px;
  `}
`;
```

**Review by michael-s-molina — COMMENTED** [review]
We can adjust the margins to our theme multiples (12px) 😉

**Review by michael-s-molina — APPROVED** [review]
LGTM. Thanks for the contribution!

**Pull request merged**